addons/mail/static/src/components/thread_preview/thread_preview_tests.js

```diff
@@ -66,6 +66,11 @@ QUnit.test('mark as read', async function (assert) {
         message_unread_counter: 1,
         model: 'mail.channel',
     });
+    this.env.models['mail.message'].create({
+        body:'lol',
```

Comment:
lint: space after `:`

Reply:
note: ideally this test should also test the change of state in the DOM due to the channel being originally unseen and then becoming seen, notably style difference and unread counter going to 0